src/platform/plugins/shared/controls/public/controls/data_controls/options_list_control/components/options_list_control.tsx

```diff
@@ -79,7 +79,7 @@ export const OptionsListControl = ({
   disableMultiValueEmptySelection?: boolean;
 }) => {
   const popoverId = useMemo(() => htmlIdGenerator()(), []);
-  const { componentApi, displaySettings } = useOptionsListContext();
+  const { componentApi, displaySettings, customStrings } = useOptionsListContext();
```

Comment:
The invalid selection label is not making sense here as the value is valid (the chart works) but is not part of the updated list.

In order to introduce a new custom label I add a customStrings property in the context. It can be a property too. Let me know what you prefer.